R/generate_retrospective_data.R

```diff
@@ -0,0 +1,99 @@
+#' Generate retrospective reporting triangles
+#'
+#' This function ingests a reporting triangle and the number of retrospective
+#'   reporting triangles we want to create, and iteratively generates the
+#'   reporting triangle that would have been available as of the maximum
+#'   reference time, and iteratively working backwards for
+#'   `n_history_uncertainty` snapshots
+#'
+#' @param triangle_for_retro Matrix of the incomplete reporting triangle
```

Comment:
This is in a function for retrospective data and there are no other inputs is it helpful to add retro here or duplicative?

Reply:
does it need to be imcomplete for this function to work? I would assume not

Reply:
I would have hoped you could share the docs for this input from another function as it should be a standard input right?